src/components/Consent/ConsentSheet.tsx

```diff
@@ -102,17 +118,26 @@ export function ConsentSheet({
               />
             </div>
 
-            <div className="mt-4 grid grid-cols-1 sm:grid-cols-2 gap-6 items-stretch">
-              {existingConsents?.results
-                ?.filter((consent) =>
-                  consent.source_attachments[0]?.name
-                    ?.toLowerCase()
-                    .includes(searchQuery.toLowerCase()),
-                ) // TODO: move this to the backend in the next iteration
-                .map((consent) => (
-                  <ConsentCard key={consent.id} consent={consent} />
-                ))}
-            </div>
+            {existingConsents?.results &&
+            existingConsents?.results?.filter((consent) =>
```

Comment:
should be able to map this to a variable instead, since we are calling it again below.